ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -813,6 +824,35 @@ export default {
         },
       },
     },
+    {
+      id: 'E12S Basic Relativity Yellow Hourglass',
+      // Orient where "Yellow" Anger Hourglass spawns
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '9824' }),
+      condition: (data, matches) => data.phase === 'basic',
+      preRun: (data, matches) => {
+        const y = parseFloat(matches.y) + 75;
+        const x = parseFloat(matches.x);
+
+        // Positions are the 8 cardinals + numerical slop on a radius=20 circle.
+        // Positions are also moved downward 75
+        // N = (0, -95), E = (20, -75), S = (0, -55), W = (-20, -75)
+        // NE = (14, -89), SE = (14, -61), SW = (-14, -61), NW = (-14, -89)
+        // Map N = 0, NE = 1, ..., NW = 7
+        const dir = Math.round(4 - 4 * Math.atan2(x, y) / Math.PI) % 8;
+
+        data.yellow = dir;
+      },
+      durationSeconds: 15,
+      infoText: (data, _, output) => output.hourglass({
+        dir1: output[data.yellow](),
+      }),
+      outputStrings: {
+        ...directions,
+        hourglass: {
+          en: 'Yellow: ${dir1}',
```

Comment:
If there's only one dir, can you call this `dir` instead of `dir1`?

Reply:
Sure thing. I was thinking this format could be used similarly for Advanced Relativity where dir1 and dir2 would be used like 'Yellow: ${dir1} / ${dir2}'.

Reply:
Yeah, that makes sense.  I think just as a style convention I was only numbering things when there was more than one.